flag-spec.yaml

```diff
@@ -12,35 +12,41 @@ flag_types:
   uint: {}
 
   string:
+    destination_pointer: true
```

Comment:
Instead of definition destination_pointer to true for everything except Generic have a type called no_destination_pointer and set that to true only for Generic. That way all these changes go away

Reply:
@nkuba I've made PR 1488 which cleans up the destination pointer for Generic so that you dont have to worry about fixing conflicts in your branch as it is very tedious and time consuming(I tried, trust me). Once that is approved you can revert your changes for flag spec and keep just the test code. Thank you so much